homeassistant/components/ezviz/camera.py

```diff
@@ -232,6 +284,24 @@ def motion_detection_enabled(self):
         """Camera Motion Detection Status."""
         return self.coordinator.data[self._idx]["alarm_notify"]
 
+    def enable_motion_detection(self):
+        """Enable motion detection in camera."""
+        try:
+            self.coordinator.ezviz_client.set_camera_defence(self._serial, 1)
+
+        except InvalidHost as err:
+            _LOGGER.error("Error enabling motion detection")
```

Comment:
We don't both log an error and raise an exception. If we're in a service handler we can just raise an exception with the message as exception argument.